app/workers/evss/document_upload.rb

```diff
@@ -14,26 +30,53 @@ class EVSS::DocumentUpload
   end
 
   def perform(auth_headers, user_uuid, document_hash)
-    document, client, uploader, file_body = nil
```

Comment:
(Blocking pending peace of mind answer) was the behavior you're (thankfully and successfully) refactoring here already tested and if so is it tested robustly? Just flagging this appears to be a relatively critical worker and it might be easy to miss something when doing this kind of refactor, as the original code is pretty hard to read.

Noting something like this would probably need to be QA'd/considered in a script for Test Rail as well but you may have already been anticipating that.

Reply:
So yes this worker has a test suite : ).  This specific line is to make these variables available outside of the Datadog::Tracing blocks.  It's no longer necessary (and an anti-pattern besides) because of the loader methods I added below. 

But to address your core question, yes this worker is tested!